package.json

```diff
@@ -41,6 +40,7 @@
     "web3": "^0.20.4"
   },
   "devDependencies": {
+    "@0xproject/abi-gen": "0.1.2",
     "@0xproject/types": "^0.1.3",
     "@types/chai": "^4.1.3",
```

Comment:
can we update this package to version 4?

Reply:
I would prefer not doing that at this time as it actually changes how the script will be run. We're doing upgrades across the board on the pre-req web3 upgrade PR https://github.com/ProjectOpenSea/wyvern-js/pull/11

Reply:
edit: saw that this refers to wyvern-js version, yes will update.